modules/mod_menu/tmpl/default.php

```diff
@@ -16,36 +16,38 @@
 	$id = ' id="' . $tagId . '"';
 }
 
+$pathSize    = count($path);
```

Comment:
Maybe also calling $pathReserve (note 'reserve' instead of 'reverse') $pathFlipped or $pathFlip?